cmake/json.cmake.in

```diff
@@ -0,0 +1,6 @@
+{
+    "DRISHTI_FACE_INNER_DETECT": "@DRISHTI_FACE_INNER_DETECT@",
+    "DRISHTI_EYE_FULL": "@DRISHTI_EYE_FULL@",
+    "DRISHTI_FACE_INNER": "@DRISHTI_FACE_INNER@",
+    "FACE_MEAN_5_POINT": "@FACE_MEAN_5_POINT@"
+}
```

Comment:
This looks good.  In this context I think the following might be better:

This looks good.  For the more public json index, I think the previous names (I used) were not very descriptive.  Let's use the following keys/labels.  I think lower case tags may be more in line with the standard conventions.
```
{
     "face_detector": "@DRISHTI_FACE_INNER_DETECT@",
     "eye_model_regressor": "@DRISHTI_EYE_FULL@",
     "face_landmark_regressor": "@DRISHTI_FACE_INNER@",
     "face_detector_mean": "@FACE_MEAN_5_POINT@"
 }
```